src/lib/legacy.js

```diff
@@ -1414,9 +1414,19 @@ export function importImage(name) {
 export function getTwitterIntentURL(text, url = "", hashtag = "") {
   let finalURL = "https://twitter.com/intent/tweet?text=";
   if (text.length > 0) {
-    finalURL += encodeURIComponent(text.replace(/[\r\n]+/g, " "))
-      .replace(/\*%7C/g, "*|URL:")
-      .replace(/%7C\*/g, "|*");
+    if (Array.isArray(text)) {
+      finalURL += text
+        .map((t) =>
+          encodeURIComponent(t.replace(/[\r\n]+/g, " "))
+            .replace(/\*%7C/g, "*|URL:")
```

Comment:
Maybe create a silmple separated function like const formatText = (t) => t.replace(...)?

Just to improve readability

Reply:
or we can remove `if` and use `(Array.isArray(text) ? text : [text]).map( ... )`